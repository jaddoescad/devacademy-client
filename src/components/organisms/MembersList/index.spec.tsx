import React from 'react'
import { render } from '@testing-library/react'
import { MembersList } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('@dev/graphql')

describe(`${MembersList.name}`, () => {
  test('Snapshot', () => {
    const component = render(<MembersList membersList={[{ propertyAddress: 'property-address', percentage: 28 }]} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
