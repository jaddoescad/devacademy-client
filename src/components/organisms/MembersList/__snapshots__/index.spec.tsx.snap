// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MembersList Snapshot 1`] = `
<body>
  <div>
    <div>
      <div
        style="font-size: 18px; line-height: 24px; margin: 0px 0px 32px 0px;"
      >
        Members
      </div>
      <div>
        <div
          style="max-width: 961px;"
        >
          <div
            class="ant-row"
          >
            <div
              class="ant-col ant-col-16"
            >
              <div
                style="font-size: 24px; line-height: 32px;"
              >
                property-address
              </div>
            </div>
            <div
              class="ant-col ant-col-8"
            >
              <div
                style="font-size: 24px; line-height: 32px;"
              >
                28
                %
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
